eldritch/events.py

```diff
@@ -1199,45 +1199,27 @@ def __init__(self, character):
     super().__init__([ForceMovement(character, "Lost"), LoseTurn(character)], character)
 
 
-class BlessCurse(Event):
-
+class BlessCurse(Sequence):
```

Comment:
I wonder if you actually want to move some of the discard logic into SelfDiscardingCard class, to ensure you only have one loan/retainer? Maybe make a SingletonSelfDiscarding card subclass that those ones inherit from? Because the Blessing/Curse logic isn't the same. 

To be investigated another day.